sugar-event-calendar/includes/admin/settings.php

```diff
@@ -599,6 +608,38 @@ function display_subsection() {
 <?php
 }
 
+/**
+ * Output the "Editing" subsection.
+ *
+ * @since 2.1.0
+ */
+function editing_subsection() {
+
+	// Get the settings
+	$type = get_option( 'sc_editor_type', 'classic' ); ?>
+
+	<table class="form-table">
+		<tbody>
+			<tr valign="top">
+				<th scope="row" valign="top">
+					<label for="sc_editor_type"><?php esc_html_e( 'Editor Type', 'sugar-calendar' ); ?></label>
+				</th>
+				<td>
+					<select name="sc_editor_type" id="sc_editor_type" class="sc-select-chosen">
+						<option value="block" <?php selected( $type, 'block' ); ?>><?php esc_html_e( 'Blocks', 'sugar-calendar' ); ?></option>
+						<option value="classic" <?php selected( $type, 'classic' ); ?>><?php esc_html_e( 'Classic', 'sugar-calendar' ); ?></option>
```

Comment:
My only suggestion would be to use `Block Editor` and `Classic Editor` to match things like the "Classic Editor" plugin. "Blocks" might also lead people to believe there are Sugar Calendar-specific blocks that can be used as well.